Phonebook.Frontend/src/app/shared/pipes/telephone.pipe.ts

```diff
@@ -6,6 +6,6 @@ import { Pipe, PipeTransform } from '@angular/core';
 export class TelephonePipe implements PipeTransform {
   // TODO: #579
   public transform(telephoneNumber: null | string): string {
-    return telephoneNumber == null ? '' : telephoneNumber.replace(/[^+0-9]/g, '');
+    return telephoneNumber == null ? 'No phone number' : telephoneNumber.replace(/[^+0-9]/g, '');
```

Comment:
Please use the translation service as described here: https://t-systems-mms.github.io/phonebook/pages/development-guides/frontend/translation-guide.html

Reply:
So that german users will get the message in german.

Reply:
Also I am still thinking about a more general message that we could also reuse for other fields (e.g. Team assistant), maybe "not know" @mschwrdtnr @paule96 what do you think?

Reply:
Of you want to use a general message. I would suggest "Not available" and in german "nicht verfügbar"

Reply:
A more generally one could be just "unknown" - "unbekannt"